tools/app-provisioning-tool/README.md

```diff
@@ -1,15 +1,11 @@
 # msidentity-app-sync 
 Command line tool that creates Microsoft identity platform applications in a tenant (AAD or B2C) and updates the configuration code of your ASP.NET Core applications (mvc, webapp, blazorwasm, blazorwasm hosted, blazorserver). The tool can also be used to update code from an existing AAD/AAD B2C application.
 
+
+
 ## Installing/Uninstalling the tool
 
-1. Build the repository and create the NuGet package (from the `tools\app-provisioning-tool` folder):
- 
-   ```Shell
-   dotnet pack
-   ```
-   
-2. Run the following in a developer command prompt in the `tools\app-provisioning-tool` folder:
+Run the following in a developer command prompt in the `tools\app-provisioning-tool` folder:
```

Comment:
this references the app-provisioning-tool\nupkg folder, which won't contain the tool if the developer has not added ran dotnet pack?

Or do you mean it will take NuGet.org first?